src/pages/_app.tsx

```diff
@@ -16,6 +16,29 @@ export default function EducatorPortal({ Component, pageProps }: AppProps) {
       <Head>
         <title>MongoDB Educator Center</title>
         <link rel="icon" href="/academia/favicon.ico" />
+        {/* Open Graph Default Metadata (Used by Facebook, LinkedIn) */}
+        <meta property="og:title" content="MongoDB Educator Center" />
+        <meta
+          property="og:description"
+          content="Explore free resources for educators crafted by MongoDB experts to prepare learners with in-demand database skills and knowledge."
+        />
+        <meta property="og:type" content="website" />
+        <meta property="og:url" content="https://www.mongodb.com/academia" />
+        <meta
+          property="og:image"
+          content="https://www.mongodb.com/academia/desk.svg"
```

Comment:
I used one of the existing hero images in the `/public` folder until Sarah or Liz Hill confirm what should be the preview image.